far/filelist.cpp

```diff
@@ -634,6 +634,20 @@ void FileList::CorrectPosition()
 		m_CurTopFile=m_CurFile-m_Stripes*m_Height+1;
 }
 
+void FileList::HorizontalScroll(const bool LeftKeyPressed)
+{
+	if (LeftKeyPressed == Global->Opt->VMenu.HorizontalArrowsMoveContents)
```

Comment:
Should FileList behaviour depend on menu settings?

Reply:
Of course, it should not. I will add another `far:config` for FileList. Unfortunately, I could not get back to this task for so long that I forgot what I did at the beginning. 🙁